src/utils/components/MutedText/MutedText.tsx

```diff
@@ -1,9 +1,13 @@
-import React, { FC } from 'react';
+import React, { FC, ReactNode } from 'react';
 
-type MutedTextProps = { text: string };
+type MutedTextProps = { content: ReactNode | string; useSpan?: boolean };
 
-const MutedText: FC<MutedTextProps> = ({ text }) => {
-  return <div className="text-muted">{text}</div>;
+const MutedText: FC<MutedTextProps> = ({ content, useSpan }) => {
+  return useSpan ? (
+    <span className="text-muted">{content}</span>
+  ) : (
+    <div className="text-muted">{content}</div>
+  );
```

Comment:
i would have change the useSpan to span as it seems misleading to me , feels like an hook name

the difference between div to span is that span will be in the same line and div will break to a new line.
this result can be achieve using CSS classname as well. if u prefer that way of div vs span its fine by me, just change useSpan to span or isSpan , you can also consider just adding a CSS using classname package, and isSpan as the boolean to add the CSS or not

Reply:
Done!